modules/ROOT/pages/db-server-cookbook.adoc

```diff
@@ -99,7 +102,7 @@ The MUnit DB server artifact in your POM file must have the `test` scope.
 
 == Define the MUnit DB Server
 
-Define the database server using the *MUnit DB Server Config* from the *Global Elements* in your canvas.
+Define the database server by creating the *global-configs.xml* file.
```

Comment:
Something like 
"Define the database server by creating the *global-configs.xml* file and defining MUnit DB Server Config".  sounds better. And in the next paragraph I'd write just "define the *MUnit DB Server Config* element as follows:"

Reply:
Let me work on the wording! 
Thanks for your comment!